### ISSUE TITLE
[bug] Error "Target class [App\Actions\Socialstream\GenerateRedirectForProvider] does not exist." on fresh install

### ISSUE THREAD

**Issue by CamKem** [joelbutcher/socialstream#260]
**Describe the bug**
I did a fresh install yesterday to try out the Laravel 10 integration and ran into an error.
"Target class [App\Actions\Socialstream\GenerateRedirectForProvider] does not exist."

**To Reproduce**
1. Install new Laravel 10 application
2. composer require joelbutcher/socialstream
3. php artisan socialstream:install --stack=inertia
4. set up database to match the .env config.
5. php artisan migrate
6. npm install && npm run dev
7. set up github config details in services.php
8. view in the browser
9. navigate to register, click github
10. See Error.

**Expected behavior**
I have never seen any reference to a class GenerateRedirectForProvider, I did a search all files in the project for it & didn't have any hits. I expected the 

**Screenshots**
Here is a share of the laravel error page - "https://flareapp.io/share/LPdN0BQ5#F65"

**Environment context**
- Socialstream version: 4.1 - the default version installed with composer.
- Jetstream stack: Inertia / Livewire
- Laravel version: 10.2.0
- PHP version: 8.2.1

**Comment by joelbutcher** [comment]
@CamKem ive created a PR for this with #261 can you give this a quick test and confirm this is working for you? Thanks!

**Comment by joelbutcher** [comment]
@CamKem fixed in 4.1.1

**Comment by CamKem** [comment]
@joelbutcher I just gave it a try - did a fresh install, fresh github Oauth app, installed the inertia stack. Then set up the config as per instruction. When I do load up vite, open the site in the browser & try to register with github I get this error.
```
Client error: `GET https://api.github.com/user` resulted in a `401 Unauthorized` response: {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest"}
```
[Here is a share of the full error](https://flareapp.io/share/17xRGw8m#F69)

Please help my diagnose what is wrong here, I am unaware of how to fix this.
